nengo_viz/components/value.py

```diff
@@ -1,6 +1,7 @@
 import nengo
 import numpy as np
 import struct
+import collections
```

Comment:
I looked at it, and thought "I really shouldn't comment on that", then I realized if I saw that later in the code I would be annoyed, and fixing it now would save time for me and all the people who would be annoyed by it later.